helm/archive-node/templates/archive.yaml

```diff
@@ -125,10 +125,14 @@ spec:
           mountPath: /root/.mina-config
         {{- if .Values.mina.runtimeConfig }}
         - name: daemon-config
-          mountPath: "/config/"
+          mountPath: "/root/config/"
         {{- end }}
         - name: actual-libp2p
           mountPath: /root/libp2p-keys
+        {{ if $.Values.persist_working_dir }}
```

Comment:
Why persist all of the user directory for the root user? We only need to persist the `.mina-config`.

Reply:
Actually in this PR i wanted to satisfy two requirements. One is to preserve `.mina-config` , second is about preserving coverage files '*.bisect' that are generated if mina image is built with instrumentation and we stop mina process. As a result bunch of coverage files will be dumped into /root folder. What's more we also wanted to preserve log files which are also dumpe into root folder. As a result the simplest method is to persist entire root folder

Reply:
The log files are actually all in `.mina-config`. The log files in the root directory is just the output of the daemon, which is a subset of the content in `.mina-config`. If you want to get the bisect files, I would recommend trying to log them to a different directory if possible (one way to do this would be to update which directory the mina process is run from, as I think the working directory is the default dump location for bisect).

Reply:
Ok, roger! what about `/root/libp2p-keys/key` should we move them also under `.mina-config` ?